web-page-gen.rkt

```diff
@@ -74,6 +80,135 @@
           (parse-style (extract-style sexp))))
        lof-sexp))
 
+
+;; Currently does not do escaping ... do not use quotes inside text
```

Comment:
So does this mean we can't have ```Create paragraph "This text has a quoted string: "quoted string""``` ?

Reply:
Yeah. I assumed we were gonna use "This text has a quoted string: \"quoted string\"", since I'm not sure it's possible to not escape them